messaging/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+class Contact(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="contacts")
```

Comment:
As i pointed out to this in the scrum on Friday, this shouldn't be a Foreign key to the User model (which django provides), we use our custom model which is Profile (accounts/models.py).
This should be a foreign key to that table instead of User.

Reply:
Same holds for other places where User is used in this model file.